File: 백준/Bronze/3058. 짝수를 찾아라/짝수를 찾아라.c
#include <stdio.h>

int main(void) {
	int n, sum, min, num;
	scanf("%d", &n);
	for (int i = 0; i < n; i++) {
		sum = 0;
		min = 100;
		for (int j = 0; j < 7; j++) {
			scanf("%d", &num);
			if (num % 2 == 0) {
				sum += num;
				if (num < min)
					min = num;
			}
		}
		printf("%d %d\n", sum, min);
	}
	return 0;
}

File: 백준/Bronze/3058. 짝수를 찾아라/짝수를 찾아라.py
n = int(input())
for _ in range(n):
    nums = list(map(int, input().split()))
    nums = [a for a in nums if a % 2 == 0]
    Min = 100
    Sum = 0
    for i in nums:
        Sum += i
        if i < Min:
            Min = i
    print(Sum, Min)